project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/js/main.js

```
@@ -0,0 +1,218 @@
+            top: section.offsetTop - parseInt(scrollMarginTop),
+            behavior: 'smooth'
+          });
+        }, 100);
+      }
+    }
+  });
+
+  /**
+   * Navmenu Scrollspy
+   */
+  let navmenulinks = document.querySelectorAll('.navmenu a');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'navmenulinks' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)